NitroxServer/GameLogic/Players/PlayerData.cs

```diff
@@ -194,6 +212,9 @@ public class PersistedPlayerData
             [ProtoMember(6)]
             public string SubRootGuid { get; set; }
 
+            [ProtoMember(7)]
+            public Perms Permissions { get; set; }
```

Comment:
Is this nullable?  Should we assign a default value if the serializer doesn't have one from the file (backwards compatibility)